tests/test_data.py

```diff
@@ -14,22 +14,40 @@
 DEFAULT_CONTEXT_PATH = os.path.join(schema_dir, "sssom.context.jsonld")
 
 
-def get_test_file(filename):
+def get_test_file(filename: TextIO) -> str:
+    """Get test file.
+
+    :param filename: Filename
+    :type filename: TextIO
+    :return: File path
+    :rtype: str
+    """
     return os.path.join(test_data_dir, filename)
 
 
 def ensure_test_dir_exists():
+    """Create directory if not already present."""
```

Comment:
Which directory?

Reply:
Tbh this shouldn't even be a function - this ensure should be in the module code

Reply:
Fixed in b47274d